.github/workflows/pr-test.yml

```diff
@@ -52,6 +54,11 @@ jobs:
             User git
             IdentityFile ~/.ssh/pipeline_config_deploy_key
 
+            Host warp.github.com
+            HostName github.com
+            User git
+            IdentityFile ~/.ssh/warp_deploy_key
+
```

Comment:
Just to record what I have done to create the deploy_keys for future:
1. Go to https://github.com/broadinstitute/warp/settings/keys and create a new `ci_read_only_key` by generating a SSH key locally. Paste the content of `.pub` there.
2. In WFL repo, create a new `secret` and paste the content of the private key there.